docs/source/_templates/index.html

```diff
@@ -83,9 +83,9 @@ <h2 style="margin-bottom: 10 px">{%trans%}Project Documentation{%endtrans%}</h2>
          <a href="https://github.com/jupyter/jupyter-js-notebook">{%trans%}jupyter-js-notebook{%endtrans%}</a><br/>
          <a href="https://github.com/jupyter/jupyter-js-phosphide">{%trans%}jupyter-js-phosphide{%endtrans%}</a><br/>
          <a href="https://github.com/jupyter/jupyter-js-plugins">{%trans%}jupyter-js-plugins{%endtrans%}</a><br/>
-         <a href="http://jupyter.org/jupyter-js-services/">{%trans%}jupyter-js-services{%endtrans%}</a><br/>
-         <a href="http://jupyter.org/jupyter-js-ui/">{%trans%}jupyter-js-ui{%endtrans%}</a><br/>
-         <a href="http://jupyter.org/jupyter-js-utils/">{%trans%}jupyter-js-utils{%endtrans%}</a></p>
+         <a href="https://github.com/jupyterlab/services">{%trans%}jupyter-js-services{%endtrans%}</a><br/>
+         <a href="https://github.com/jupyter-attic/jupyter-js-ui">{%trans%}jupyter-js-ui{%endtrans%}</a><br/>
+         <a href="https://github.com/jupyter-attic/jupyter-js-utils">{%trans%}jupyter-js-utils{%endtrans%}</a></p>
```

Comment:
@minrk I've gone ahead and removed all the repo links that point to Jupyter-attic